userdocs/docs/getting-started/basics/interacting-with-the-service.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 5
+---
+
+# Interacting with the service
```

Comment:
we could add extra section here about simply building a client using the grpc methods

Reply:
That sounds more like a "write documentation" than "Initial creation of documentation site"

Reply:
haha okay